UnityProject/Assets/Scripts/Lighting/LightMountStates.cs

```diff
@@ -241,19 +249,25 @@ private void OnDamageReceived(DamageInfo arg0)
 
 	private void SyncLightState(LightMountState oldState, LightMountState newState)
 	{
-		state = newState;
+		//state = newState;
 		ChangeLightState(newState);
 	}
 
 	public override void OnStartClient()
 	{
-		SyncLightState(state, state);
```

Comment:
Calling the syncvar hook in OnStartClient is actually the correct way to do it. SyncVars are loaded with the correct value from server in the OnStartClient hook but the SyncVar hook is not called. You need to do it manually here.

Reply:
That actually makes sense. The docs state exactly the same thing with the value being the latest from the server prior to OnStartClient but never actually runs it. Makes sense that hooks won't start running until the client has started.